config/routes.rb

```diff
@@ -22,15 +22,25 @@
       resource :extracts
       resources :extracts, only: [:index]
     end
-    
+
     resources :reducers
     resources :subject_reductions, param: :reducer_key
     resources :user_reductions, param: :reducer_key
 
+    resources :subject_rules do
+      resources :subject_rule_effects
+    end
+
+    resources :user_rules do
+      resources :user_rule_effects
+    end
+
     resources :subjects, only: [:show] do
       resources :subject_reductions, only: [:index]
+      resources :data_requests
```

Comment:
I can't tell what prompted the addition of this line? The data requests controller doesn't seem to support scoping onto a single subject in a workflow.